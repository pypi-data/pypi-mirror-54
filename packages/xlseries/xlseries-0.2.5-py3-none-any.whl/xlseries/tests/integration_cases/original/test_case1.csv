;IPC-CqP;Var. Anual;;
ene-93;48,46;;;
feb-93;48,81;;;
mar-93;49,18;;;
abr-93;49,69;;;
may-93;50,33;;;
jun-93;50,69;;;
jul-93;50,86;;;
ago-93;50,86;;;
sept-93;51,28;;;
oct-93;51,57;;;
nov-93;51,60;;;
dic-93;51,60;;;
ene-94;51,65;;;
feb-94;51,65;;;
mar-94;51,72;;;
abr-94;51,84;;;
may-94;52,02;;;
jun-94;52,23;;;
jul-94;52,71;;;
ago-94;52,82;;;
sept-94;53,18;;;
oct-94;53,35;;;
nov-94;53,47;;;
dic-94;53,58;;;
ene-95;54,25;;;
feb-95;54,25;;;
mar-95;54,01;;;
abr-95;54,25;;;
may-95;54,27;;;
jun-95;54,15;;;
jul-95;54,37;;;
ago-95;54,24;;;
sept-95;54,33;;;
oct-95;54,52;;;
nov-95;54,39;;;
dic-95;54,45;;;
ene-96;54,61;;;
feb-96;54,43;;;
mar-96;54,14;;;
abr-96;54,14;;;
may-96;54,09;;;
jun-96;54,09;;;
jul-96;54,39;;;
ago-96;54,34;;;
sept-96;54,44;;;
oct-96;54,72;;;
nov-96;54,63;;;
dic-96;54,48;;;
ene-97;54,73;;;
feb-97;54,94;;;
mar-97;54,67;;;
abr-97;54,49;;;
may-97;54,44;;;
jun-97;54,57;;;
jul-97;54,69;;;
ago-97;54,78;;;
sept-97;54,75;;;
oct-97;54,67;;;
nov-97;54,56;;;
dic-97;54,65;;;
ene-98;55,00;;;
feb-98;55,19;;;
mar-98;55,12;;;
abr-98;55,13;;;
may-98;55,09;;;
jun-98;55,19;;;
jul-98;55,36;;;
ago-98;55,38;;;
sept-98;55,36;;;
oct-98;55,16;;;
nov-98;55,03;;;
dic-98;55,02;;;
ene-99;55,28;;;
feb-99;55,19;;;
mar-99;54,77;;;
abr-99;54,72;;;
may-99;54,45;;;
jun-99;54,45;;;
jul-99;54,55;;;
ago-99;54,34;;;
sept-99;54,24;;;
oct-99;54,23;;;
nov-99;54,06;;;
dic-99;54,02;;;
ene-00;54,48;;;
feb-00;54,48;;;
mar-00;54,19;;;
abr-00;54,13;;;
may-00;53,92;;;
jun-00;53,82;;;
jul-00;54,06;;;
ago-00;53,94;;;
sept-00;53,86;;;
oct-00;53,95;;;
nov-00;53,69;;;
dic-00;53,63;;;
ene-01;53,67;;;
feb-01;53,55;;;
mar-01;53,65;;;
abr-01;54,01;;;
may-01;54,04;;;
jun-01;53,65;;;
jul-01;53,48;;;
ago-01;53,29;;;
sept-01;53,25;;;
oct-01;53,01;;;
nov-01;52,84;;;
dic-01;52,80;;;
ene-02;54,01;;;
feb-02;55,70;;;
mar-02;57,91;;;
abr-02;63,92;;;
may-02;66,49;;;
jun-02;68,90;;;
jul-02;71,09;;;
ago-02;72,76;;;
sept-02;73,74;;;
oct-02;73,90;;;
nov-02;74,28;;;
dic-02;74,42;;;
ene-03;75,40;;;
feb-03;75,83;;;
mar-03;76,27;;;
abr-03;76,31;;;
may-03;76,02;;;
jun-03;75,95;;;
jul-03;76,29;;;
ago-03;76,31;;;
sept-03;76,34;;;
oct-03;76,79;;;
nov-03;76,98;;;
dic-03;77,14;;;
ene-04;77,47;;;
feb-04;77,54;;;
mar-04;78,00;;;
abr-04;78,67;;;
may-04;79,25;;;
jun-04;79,70;;;
jul-04;80,06;;;
ago-04;80,34;;;
sept-04;80,84;;;
oct-04;81,16;;;
nov-04;81,17;;;
dic-04;81,84;;;
ene-05;83,06;;;
feb-05;83,85;;;
mar-05;85,14;;;
abr-05;85,56;;;
may-05;86,07;;;
jun-05;86,86;;;
jul-05;87,73;;;
ago-05;88,12;;;
sept-05;89,14;;;
oct-05;89,84;;;
nov-05;90,92;;;
dic-05;91,94;;;
ene-06;93,11;;;
feb-06;93,48;;;
mar-06;94,60;;;
abr-06;95,52;;;
may-06;95,97;;;
jun-06;96,44;;;
jul-06;97,03;;;
ago-06;97,58;;;
sept-06;98,45;;;
oct-06;99,30;;;
nov-06;100,00;;;
dic-06;101,77;10,7%;;
ene-07;103,18;10,8%;;
feb-07;104,02;11,3%;;
mar-07;105,03;11,0%;;
abr-07;107,09;12,1%;;
may-07;109,49;14,1%;;
jun-07;112,24;16,4%;;
jul-07;115,16;18,7%;;
ago-07;119,83;22,8%;;
sept-07;122,86;24,8%;;
oct-07;125,85;26,7%;;
nov-07;126,83;26,8%;;
dic-07;128,16;25,9%;;
ene-08;129,97;26,0%;;
feb-08;133,14;28,0%;;
mar-08;137,48;30,9%;;
abr-08;142,34;32,9%;;
may-08;144,05;31,6%;;
jun-08;146,78;30,8%;;
jul-08;148,91;29,3%;;
ago-08;150,55;25,6%;;
sept-08;152,44;24,1%;;
oct-08;154,37;22,7%;;
nov-08;155,57;22,7%;;
dic-08;156,06;21,8%;;
ene-09;157,11;20,9%;;
feb-09;158,14;18,8%;;
mar-09;160,26;16,6%;;
abr-09;161,95;13,8%;;
may-09;162,96;13,1%;;
jun-09;163,80;11,6%;;
jul-09;165,91;11,4%;;
ago-09;168,54;11,9%;;
sept-09;170,82;12,1%;;
oct-09;173,74;12,5%;;
nov-09;175,85;13,0%;;
dic-09;179,50;15,0%;;
ene-10;182,48;16,2%;;
feb-10;188,47;19,2%;;
mar-10;193,99;21,0%;;
abr-10;197,69;22,1%;;
may-10;200,77;23,2%;;
jun-10;204,00;24,5%;;
jul-10;207,52;25,1%;;
ago-10;210,60;25,0%;;
sept-10;214,08;25,3%;;
oct-10;219,70;26,5%;;
nov-10;224,08;27,4%;;
dic-10;227,81;26,9%;;
ene-11;232,38;27,3%;;
feb-11;235,61;25,0%;;
mar-11;240,64;24,0%;;
abr-11;245,86;24,4%;;
may-11;250,74;24,9%;;
jun-11;255,34;25,2%;;
jul-11;260,68;25,6%;;
ago-11;265,47;26,1%;;
sept-11;270,77;26,5%;;
oct-11;273,66;24,6%;;
nov-11;277,73;23,9%;;
dic-11;282,95;24,2%;;
ene-12;286,95;23,5%;;
feb-12;292,10;24,0%;;
mar-12;297,93;23,8%;;
abr-12;304,93;24,0%;;
may-12;310,07;23,7%;;
jun-12;315,05;23,4%;;
jul-12;321,75;23,4%;;
ago-12;328,41;23,7%;;
sept-12;334,01;23,4%;;
oct-12;339,09;23,9%;;
nov-12;345,61;24,4%;;
dic-12;351,87;24,4%;;
ene-13;358,26;24,9%;;
feb-13;364,03;24,6%;;
mar-13;369,70;24,1%;;
abr-13;375,80;23,2%;;
may-13;381,88;23,2%;;
jun-13;389,09;23,5%;;
jul-13;398,72;23,9%;;
ago-13;406,75;23,9%;;
sept-13;414,06;24,0%;;
oct-13;425,59;25,5%;;
nov-13;437,31;26,5%;;
dic-13;448,59;27,5%;;
ene-14;469,16;31,0%;;Revisado
feb-14;491,82;35,1%;Provisorio;Revisado
mar-14;508,19;37,5%;Provisorio;
;;;;
Se puede utilizar citando la fuente;;;;
Fuente: Cosas que Pasan en base a Direcciones Provinciales de Estad√≠stica;;;;
http://elhombrecitodelsombrerogris.blogspot.com/;;;;
;;;;