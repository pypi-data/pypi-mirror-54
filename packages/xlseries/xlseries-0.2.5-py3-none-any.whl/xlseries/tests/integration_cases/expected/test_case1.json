{"IPC-CqP":{"725846400000":48.45688507,"728524800000":48.81083389,"730944000000":49.17781936,"733622400000":49.69188897,"736214400000":50.33120549,"738892800000":50.69324133,"741484800000":50.8563183,"744163200000":50.86447328,"746841600000":51.2836393,"749433600000":51.57353754,"752112000000":51.60302006,"754704000000":51.5962469,"757382400000":51.64820092,"760060800000":51.64637738,"762480000000":51.71836774,"765158400000":51.84426025,"767750400000":52.02395299,"770428800000":52.22527908,"773020800000":52.70744233,"775699200000":52.81618673,"778377600000":53.17766758,"780969600000":53.34807271,"783648000000":53.46868714,"786240000000":53.58496357,"788918400000":54.25261865,"791596800000":54.2511349,"794016000000":54.00739158,"796694400000":54.25438557,"799286400000":54.26579121,"801964800000":54.154034,"804556800000":54.37374277,"807235200000":54.24197187,"809913600000":54.33116698,"812505600000":54.51587729,"815184000000":54.39135527,"817776000000":54.44648067,"820454400000":54.60970488,"823132800000":54.43224344,"825638400000":54.13858484,"828316800000":54.13941166,"830908800000":54.09112738,"833587200000":54.09253185,"836179200000":54.38581668,"838857600000":54.34369393,"841536000000":54.4423692,"844128000000":54.71680696,"846806400000":54.63182527,"849398400000":54.4760538,"852076800000":54.73048921,"854755200000":54.94093302,"857174400000":54.67009704,"859852800000":54.48940759,"862444800000":54.44419275,"865123200000":54.56815978,"867715200000":54.68941982,"870393600000":54.77941909,"873072000000":54.75318724,"875664000000":54.6674127,"878342400000":54.56176039,"880934400000":54.65472717,"883612800000":54.99827839,"886291200000":55.19004503,"888710400000":55.12071637,"891388800000":55.12693454,"893980800000":55.08712238,"896659200000":55.19228765,"899251200000":55.36478815,"901929600000":55.37596727,"904608000000":55.35884181,"907200000000":55.15631513,"909878400000":55.0253484,"912470400000":55.01789565,"915148800000":55.27725802,"917827200000":55.18845934,"920246400000":54.7740957,"922924800000":54.72045405,"925516800000":54.45103387,"928195200000":54.44772657,"930787200000":54.54892762,"933465600000":54.343592,"936144000000":54.2353233,"938736000000":54.22683985,"941414400000":54.05543801,"944006400000":54.02182137,"946684800000":54.47843234,"949363200000":54.48066363,"951868800000":54.19281546,"954547200000":54.13187963,"957139200000":53.9214698,"959817600000":53.82177516,"962409600000":54.05552862,"965088000000":53.93922954,"967766400000":53.85639985,"970358400000":53.95291178,"973036800000":53.68769368,"975628800000":53.62767529,"978307200000":53.67080607,"980985600000":53.54854932,"983404800000":53.65057719,"986083200000":54.0092038,"988675200000":54.04415696,"991353600000":53.65434887,"993945600000":53.47885821,"996624000000":53.28758993,"999302400000":53.24724543,"1001894400000":53.01233667,"1004572800000":52.83847701,"1007164800000":52.7983024,"1009843200000":54.00876207,"1012521600000":55.70489616,"1014940800000":57.90829271,"1017619200000":63.9241768,"1020211200000":66.48801218,"1022889600000":68.89622334,"1025481600000":71.09308458,"1028160000000":72.75805984,"1030838400000":73.7408483,"1033430400000":73.90202244,"1036108800000":74.27828419,"1038700800000":74.41737192,"1041379200000":75.39914101,"1044057600000":75.82591834,"1046476800000":76.26855258,"1049155200000":76.31068665,"1051747200000":76.01801345,"1054425600000":75.95288686,"1057017600000":76.29018593,"1059696000000":76.30876117,"1062374400000":76.33900256,"1064966400000":76.78899893,"1067644800000":76.97837571,"1070236800000":77.14181512,"1072915200000":77.46608887,"1075593600000":77.54401424,"1078099200000":78.00409108,"1080777600000":78.67336583,"1083369600000":79.24817872,"1086048000000":79.69670267,"1088640000000":80.0640166,"1091318400000":80.33902068,"1093996800000":80.84462949,"1096588800000":81.16459923,"1099267200000":81.16641145,"1101859200000":81.84497381,"1104537600000":83.06108534,"1107216000000":83.84679509,"1109635200000":85.14287073,"1112313600000":85.56036045,"1114905600000":86.07435077,"1117584000000":86.86266559,"1120176000000":87.73490875,"1122854400000":88.11819286,"1125532800000":89.14300212,"1128124800000":89.8395734,"1130803200000":90.92339302,"1133395200000":91.93563002,"1136073600000":93.1085881,"1138752000000":93.47760099,"1141171200000":94.60434751,"1143849600000":95.52438792,"1146441600000":95.97143944,"1149120000000":96.43616009,"1151712000000":97.03124717,"1154390400000":97.57706457,"1157068800000":98.45463112,"1159660800000":99.29731248,"1162339200000":100.0,"1164931200000":101.77210724,"1167609600000":103.1825974,"1170288000000":104.01509547,"1172707200000":105.0349722,"1175385600000":107.08974927,"1177977600000":109.49377193,"1180656000000":112.2412531,"1183248000000":115.15958546,"1185926400000":119.83169121,"1188604800000":122.86316457,"1191196800000":125.85327848,"1193875200000":126.83152654,"1196467200000":128.15612487,"1199145600000":129.9654575,"1201824000000":133.13592268,"1204329600000":137.48126591,"1207008000000":142.34105336,"1209600000000":144.05034149,"1212278400000":146.77908128,"1214870400000":148.91144375,"1217548800000":150.55430639,"1220227200000":152.44273701,"1222819200000":154.36550503,"1225497600000":155.57259717,"1228089600000":156.05693449,"1230768000000":157.10570291,"1233446400000":158.1434391,"1235865600000":160.26383352,"1238544000000":161.95094117,"1241136000000":162.96053841,"1243814400000":163.79817635,"1246406400000":165.90620013,"1249084800000":168.53505649,"1251763200000":170.81950046,"1254355200000":173.73629738,"1257033600000":175.85022647,"1259625600000":179.50254135,"1262304000000":182.47995051,"1264982400000":188.46544884,"1267401600000":193.99204512,"1270080000000":197.68536599,"1272672000000":200.77256199,"1275350400000":203.99947141,"1277942400000":207.52399645,"1280620800000":210.60474682,"1283299200000":214.08168072,"1285891200000":219.70177339,"1288569600000":224.07940012,"1291161600000":227.81249171,"1293840000000":232.37512966,"1296518400000":235.6065168,"1298937600000":240.64474082,"1301616000000":245.86492153,"1304208000000":250.73504644,"1306886400000":255.33647468,"1309478400000":260.67974052,"1312156800000":265.47442336,"1314835200000":270.77050328,"1317427200000":273.65619385,"1320105600000":277.73027068,"1322697600000":282.95249305,"1325376000000":286.94590043,"1328054400000":292.10309863,"1330560000000":297.92722674,"1333238400000":304.92671556,"1335830400000":310.0685158,"1338508800000":315.04905266,"1341100800000":321.75202526,"1343779200000":328.41210199,"1346457600000":334.00655283,"1349049600000":339.09253503,"1351728000000":345.60875096,"1354320000000":351.86610921,"1356998400000":358.25632184,"1359676800000":364.02916542,"1362096000000":369.70395789,"1364774400000":375.79927706,"1367366400000":381.87885145,"1370044800000":389.08714403,"1372636800000":398.71969357,"1375315200000":406.7492389,"1377993600000":414.06054611,"1380585600000":425.5874021,"1383264000000":437.31358173,"1385856000000":448.58877361,"1388534400000":469.16138816,"1391212800000":491.81879986,"1393632000000":508.18640763},"Var. Anual":{"725846400000":null,"728524800000":null,"730944000000":null,"733622400000":null,"736214400000":null,"738892800000":null,"741484800000":null,"744163200000":null,"746841600000":null,"749433600000":null,"752112000000":null,"754704000000":null,"757382400000":null,"760060800000":null,"762480000000":null,"765158400000":null,"767750400000":null,"770428800000":null,"773020800000":null,"775699200000":null,"778377600000":null,"780969600000":null,"783648000000":null,"786240000000":null,"788918400000":null,"791596800000":null,"794016000000":null,"796694400000":null,"799286400000":null,"801964800000":null,"804556800000":null,"807235200000":null,"809913600000":null,"812505600000":null,"815184000000":null,"817776000000":null,"820454400000":null,"823132800000":null,"825638400000":null,"828316800000":null,"830908800000":null,"833587200000":null,"836179200000":null,"838857600000":null,"841536000000":null,"844128000000":null,"846806400000":null,"849398400000":null,"852076800000":null,"854755200000":null,"857174400000":null,"859852800000":null,"862444800000":null,"865123200000":null,"867715200000":null,"870393600000":null,"873072000000":null,"875664000000":null,"878342400000":null,"880934400000":null,"883612800000":null,"886291200000":null,"888710400000":null,"891388800000":null,"893980800000":null,"896659200000":null,"899251200000":null,"901929600000":null,"904608000000":null,"907200000000":null,"909878400000":null,"912470400000":null,"915148800000":null,"917827200000":null,"920246400000":null,"922924800000":null,"925516800000":null,"928195200000":null,"930787200000":null,"933465600000":null,"936144000000":null,"938736000000":null,"941414400000":null,"944006400000":null,"946684800000":null,"949363200000":null,"951868800000":null,"954547200000":null,"957139200000":null,"959817600000":null,"962409600000":null,"965088000000":null,"967766400000":null,"970358400000":null,"973036800000":null,"975628800000":null,"978307200000":null,"980985600000":null,"983404800000":null,"986083200000":null,"988675200000":null,"991353600000":null,"993945600000":null,"996624000000":null,"999302400000":null,"1001894400000":null,"1004572800000":null,"1007164800000":null,"1009843200000":null,"1012521600000":null,"1014940800000":null,"1017619200000":null,"1020211200000":null,"1022889600000":null,"1025481600000":null,"1028160000000":null,"1030838400000":null,"1033430400000":null,"1036108800000":null,"1038700800000":null,"1041379200000":null,"1044057600000":null,"1046476800000":null,"1049155200000":null,"1051747200000":null,"1054425600000":null,"1057017600000":null,"1059696000000":null,"1062374400000":null,"1064966400000":null,"1067644800000":null,"1070236800000":null,"1072915200000":null,"1075593600000":null,"1078099200000":null,"1080777600000":null,"1083369600000":null,"1086048000000":null,"1088640000000":null,"1091318400000":null,"1093996800000":null,"1096588800000":null,"1099267200000":null,"1101859200000":null,"1104537600000":null,"1107216000000":null,"1109635200000":null,"1112313600000":null,"1114905600000":null,"1117584000000":null,"1120176000000":null,"1122854400000":null,"1125532800000":null,"1128124800000":null,"1130803200000":null,"1133395200000":null,"1136073600000":null,"1138752000000":null,"1141171200000":null,"1143849600000":null,"1146441600000":null,"1149120000000":null,"1151712000000":null,"1154390400000":null,"1157068800000":null,"1159660800000":null,"1162339200000":null,"1164931200000":0.1069930909,"1167609600000":0.1081963491,"1170288000000":0.112727481,"1172707200000":0.1102552363,"1175385600000":0.1210723419,"1177977600000":0.1408995484,"1180656000000":0.1638917704,"1183248000000":0.1868299009,"1185926400000":0.2280723112,"1188604800000":0.2479165598,"1191196800000":0.2674389199,"1193875200000":0.2683152654,"1196467200000":0.2592460581,"1199145600000":0.259567609,"1201824000000":0.279967317,"1204329600000":0.3089094331,"1207008000000":0.3291753349,"1209600000000":0.3156030608,"1212278400000":0.3077106431,"1214870400000":0.2930877023,"1217548800000":0.2563813868,"1220227200000":0.2407521616,"1222819200000":0.2265513215,"1225497600000":0.2266082528,"1228089600000":0.2177095293,"1230768000000":0.2088266062,"1233446400000":0.1878344771,"1235865600000":0.1657139789,"1238544000000":0.1377669151,"1241136000000":0.1312749191,"1243814400000":0.1159504128,"1246406400000":0.1141265973,"1249084800000":0.1194303274,"1251763200000":0.1205486323,"1254355200000":0.125486535,"1257033600000":0.130341909,"1259625600000":0.1502375203,"1262304000000":0.1615106717,"1264982400000":0.1917373867,"1267401600000":0.2104542919,"1270080000000":0.2206496891,"1272672000000":0.2320317787,"1275350400000":0.2454318843,"1277942400000":0.2508513623,"1280620800000":0.2496198192,"1283299200000":0.2532625382,"1285891200000":0.2645703673,"1288569600000":0.2742627895,"1291161600000":0.2691324033,"1293840000000":0.2734282808,"1296518400000":0.250131089,"1298937600000":0.240487674,"1301616000000":0.2437183718,"1304208000000":0.2488511575,"1306886400000":0.2516526288,"1309478400000":0.2561426388,"1312156800000":0.2605339023,"1314835200000":0.2647999697,"1317427200000":0.2455802683,"1320105600000":0.2394279462,"1322697600000":0.2420411669,"1325376000000":0.2348391192,"1328054400000":0.2397921017,"1330560000000":0.2380375558,"1333238400000":0.2402204986,"1335830400000":0.2366381174,"1338508800000":0.2338583943,"1341100800000":0.234280902,"1343779200000":0.237076242,"1346457600000":0.2335411309,"1349049600000":0.2391188018,"1351728000000":0.2444043284,"1354320000000":0.243551896,"1356998400000":0.2485152124,"1359676800000":0.2462352064,"1362096000000":0.240920348,"1364774400000":0.2324249004,"1367366400000":0.2315950572,"1370044800000":0.2350049643,"1372636800000":0.2392142466,"1375315200000":0.2385330396,"1377993600000":0.2396779123,"1380585600000":0.2550774733,"1383264000000":0.2653429073,"1385856000000":0.274884855,"1388534400000":0.309569042,"1391212800000":0.3510422971,"1393632000000":0.3745765951}}